easybuild/easyconfigs/p/PsiCLASS/PsiCLASS-1.0.3-foss-2021b.eb

```
@@ -0,0 +1,32 @@
+easyblock = "MakeCp"
+
+name = "PsiCLASS"
+version = "1.0.3"
+
+homepage = "https://github.com/splicebox/PsiCLASS"
+description = """PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples."""
+
+toolchain = {"name": "foss", "version": "2021b"}
```

GCC should be sufficient, it doesn't look like it needs MPI/BLAS
```suggestion
toolchain = {"name": "GCC", "version": "11.2.0"}
```

---

done, thank you